src/PSDocs.Azure/docs/Azure.Template.Doc.ps1

```diff
@@ -236,5 +236,16 @@ Document 'README' {
                 $example | Code 'json'
             }
         }
+        # Add command line snippet
+        if ($PSDocs.Configuration.GetBoolOrDefault('AZURE_USE_COMMAND_LINE_SNIPPET', $True)) {
```

Comment:
If default is not generate the snippet you can set this to $False.

Reply:
Thanks @BernieWhite - have updated this.